### ISSUE TITLE
add a link to the training data

### ISSUE THREAD

**Issue by tacman** [sgoranov/image-content-moderation#1]
Alternatively, include the training data with git lfs.

It looks like the training data embeds the expected values in the filename.  I wonder if there's an alternative, like putting the training images in separate directories?

Thanks so much for releasing this!

**Comment by sgoranov** [comment]
Unfortunately, I don’t have the rights to publish the training data. However, I’ve committed an additional pre-trained model that was trained using images resized to 64x64 pixels.
Regarding the labels incorporated into the filenames, the main benefit of this approach is the ability to assign multiple labels to a single image. Other than that, there’s absolutely no problem using directories for labeling as you described.

**Comment by tacman** [comment]
OK.  Wow, the training is done on such tiny images!    

I have a ton of museum data (e.g. https://www.flickr.com/photos/museado/albums/72177720319153183/with/53882975565) that is already classified and could be used for training data for other purposes (alas, not for adult content).  Maybe I'll release a set that can be used as an example for training.

Thanks again for releasing this.  I hope the tensor pecl extension support php 8.4 soon!
